lib/client-assets.php

```diff
@@ -96,11 +96,13 @@ function wc_admin_register_script() {
 	}
 
 	// Resets lodash to wp-admin's version of lodash.
-	wp_add_inline_script(
-		WC_ADMIN_APP,
-		'_.noConflict();',
-		'after'
-	);
+	if ( 'embedded' === $entry ) {
+		wp_add_inline_script(
+			WC_ADMIN_APP,
+			'_ = _.noConflict();',
```

Comment:
I'm seeing `Uncaught TypeError: Cannot read property 'noConflict' of undefined` on non-dashboard/analytics pages.